code/game/gamemodes/clock_cult/clock_unsorted.dm

```diff
@@ -227,23 +225,77 @@ actually very simple. To translate a plain English sentence to Ratvar's tongue,
-- In the same vein, any time "ti", followed by one letter occurs, you add an apostrophe between "i" and the letter, i.e; "Ti'me"
+- Any time "th", followed by any two letters occurs, you add a grave (`) between those two letters, i.e; "Thi`s"
+- In the same vein, any time "ti", followed by one letter occurs, you add a grave (`) between "i" and the letter, i.e; "Ti`me"
 - Whereever "te" or "et" appear and there is another letter next to the e(i.e; "m"etal, greate"r"), add a hyphen between "e" and the letter, i.e; "M-etal", "Greate-r"
-- Where the word "and" appears, it is linked to the previous and following words by hyphens, i.e; "Sword-and-shield"
+- Where the word "and" appears,it is linked to all surrounding words by hyphens, i.e; "Sword-and-shield", but not "And this?"
 - Where the word "to" appears, it is linked to the following word by a hyphen, i.e; "to-use"
 - Where the word "my" appears, it is linked to the following word by a hyphen, i.e; "my-light"
-
 - Although "Ratvar" translates to "Engine" in English, the word "Ratvar" is used regardless of language as it is a proper noun.
  - The same rule applies to Ratvar's four generals: Nezbere (Armorer), Sevtug (Fright), Nzcrentr (Amperage), and Inath-Neq (Vangu-Ard), although these words can be used in proper context if one is
    not referring to the four generals and simply using the words themselves.
-
 */
+
+//Regexes used to alter english to ratvarian style
+#define RATVAR_OF_MATCH				regex("(\\w)\\s(\[oO]\[fF])","g")
+#define RATVAR_OF_REPLACEMENT 		"$1-$2"
+#define RATVAR_TH_MATCH				regex("(\[tT]\[hH]\\w)(\\w)","g")
+#define RATVAR_TH_REPLACEMENT		"$1`$2"
+#define RATVAR_TI_MATCH				regex("(\[tT]\[iI])(\\w{2})","g")
+#define RATVAR_TI_REPLACEMENT		"$1`$2"
+#define RATVAR_ET_MATCH				regex("(\[eE]\[tT])(\\w)","g")
+#define RATVAR_ET_REPLACEMENT		"-$1$2"
+#define RATVAR_TE_MATCH				regex("(\[tT]\[eE])(\\w)","g")
+#define RATVAR_TE_REPLACEMENT		"$1-$2"
+#define RATVAR_PRE_AND_MATCH		regex("(\\w)\\s(\[aA]\[nN]\[dD])","g")
+#define RATVAR_PRE_AND_REPLACEMENT	"$1-$2"
+#define RATVAR_POST_AND_MATCH		regex("(\[aA]\[nN]\[dD])\\s(\\w)","g")
+#define RATVAR_POST_AND_REPLACEMENT	"$1-$2"
+#define RATVAR_TO_MATCH				regex("(\\s)(\[tT]\[oO])\\s(\\w)","g")
+#define RATVAR_TO_REPLACEMENT		"$1$2-$3"
+#define RATVAR_MY_MATCH 			regex("(\\s)(\[mM]\[yY])\\s(\\w)","g")
+#define RATVAR_MY_REPLACEMENT		"$1$2-$3"
+
+//Regexes used to remove ratvarian styling from english
+#define REVERSE_RATVAR_HYPHEN_PRE_AND_MATCH			regex("(\\w)-(\[aA]\[nN]\[dD])","g") //specifically structured to support -emphasis-, including with -and-
+#define REVERSE_RATVAR_HYPHEN_PRE_AND_REPLACEMENT	"$1 $2"
+#define REVERSE_RATVAR_HYPHEN_POST_AND_MATCH		regex("(\[aA]\[nN]\[dD])-(\\w)","g")
+#define REVERSE_RATVAR_HYPHEN_POST_AND_REPLACEMENT	"$1 $2"
+#define REVERSE_RATVAR_HYPHEN_TO_MY_MATCH			regex("(\[tTmM]\[oOyY])-","g")
```

Comment:
yeah those shouldn't appear, which is why I used this version, it should be fine.